modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testIntegration/java/com/liferay/headless/admin/taxonomy/resource/v1_0/test/TaxonomyCategoryResourceTest.java

```diff
@@ -169,14 +170,31 @@ public void testGetTaxonomyCategory() throws Exception {
 
 		Assert.assertNull(postTaxonomyCategory.getTaxonomyCategoryUsageCount());
 
-		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
+		JSONObject jsonObject1 = HTTPTestUtil.invokeToJSONObject(
 			null,
 			"headless-admin-taxonomy/v1.0/taxonomy-categories/" +
 				getTaxonomyCategory.getId() +
 					"?nestedFields=taxonomyCategoryUsageCount",
 			Http.Method.GET);
 
-		Assert.assertNotNull(jsonObject.get("taxonomyCategoryUsageCount"));
+		Assert.assertNotNull(jsonObject1.get("taxonomyCategoryUsageCount"));
+
+		_addTaxonomyCategoryWithParentTaxonomyCategory(
+			postTaxonomyCategory.getId(), randomTaxonomyCategory());
+
+		JSONObject jsonObject2 = HTTPTestUtil.invokeToJSONObject(
```

Comment:
I think you could reuse and reassign the `jsonObject` variable instead of declaring 2 different ones. We've done it like that in [other places](https://github.com/liferay/liferay-portal/blob/0f79173a67da790949dca2049ad9127f3c8d67a9/modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APISortRelevantObjectEntryModelListenerTest.java#L75). I don't know if Brian will like it. 

You can try to forward it as it is now if you want, I don't mind.

Reply:
the case is different though 🤔 In the first request that results with jsonObject1 we check taxonomy-categories, and in the jsonObjec2 we call for its children